pangeo-notebook/environment.yml

```diff
@@ -89,7 +89,7 @@ dependencies:
  - xarray
  - xarrayutils
  - xarray-datatree
- - xarray_leaflet
+ - xarray_leaflet>=0.2.3
```

Comment:
i think this should be 0.2.2? In any case, once it's on conda-forge (see linked issue) we can add it!